diff -up samba-4.13.0rc1/source3/utils/net_util.c.4~ samba-4.13.0rc1/source3/utils/net_util.c
--- samba-4.13.0rc1/source3/utils/net_util.c.4~	2020-07-11 20:31:52.819501256 +0200
+++ samba-4.13.0rc1/source3/utils/net_util.c	2020-07-11 20:33:31.440679705 +0200
@@ -121,6 +121,11 @@ NTSTATUS connect_to_service(struct net_c
 		signing_setting = SMB_SIGNING_IPC_DEFAULT;
 	}
 
+        if(((struct functable*)c->private_data)->valid_transports == NET_TRANSPORT_RAP)
+        {
+                signing_setting = SMB_SIGNING_DEFAULT;
+        }
+
 	nt_status = cli_full_connection_creds(cli_ctx, NULL, server_name,
 					server_ss, c->opt_port,
 					service_name, service_type,
