diff -up samba-4.11.0rc2/lib/replace/system/network.h.omv~ samba-4.11.0rc2/lib/replace/system/network.h
--- samba-4.11.0rc2/lib/replace/system/network.h.omv~	2019-08-23 22:09:57.781158376 +0200
+++ samba-4.11.0rc2/lib/replace/system/network.h	2019-08-23 22:10:02.272141508 +0200
@@ -87,10 +87,6 @@
 #include <sys/uio.h>
 #endif
 
-#ifdef HAVE_STROPTS_H
-#include <stropts.h>
-#endif
-
 #ifndef HAVE_SOCKLEN_T
 #define HAVE_SOCKLEN_T
 typedef int socklen_t;
