--- samba-4.7.4/packaging/systemd/nmb.service.0001~	2017-12-30 20:52:42.498002142 +0100
+++ samba-4.7.4/packaging/systemd/nmb.service	2017-12-30 20:55:29.933989361 +0100
@@ -5,7 +5,7 @@ After=syslog.target network.target
 [Service]
 Type=notify
 NotifyAccess=all
-PIDFile=/run/nmbd.pid
+PIDFile=/run/samba/nmbd.pid
 EnvironmentFile=-/etc/sysconfig/samba
 ExecStart=/usr/sbin/nmbd --foreground --no-process-group $NMBDOPTIONS
 ExecReload=/usr/bin/kill -HUP $MAINPID
--- samba-4.7.4/packaging/systemd/smb.service.0001~	2017-12-30 20:52:42.498002142 +0100
+++ samba-4.7.4/packaging/systemd/smb.service	2017-12-30 20:55:16.966057571 +0100
@@ -5,7 +5,7 @@ After=syslog.target network.target nmb.s
 [Service]
 Type=notify
 NotifyAccess=all
-PIDFile=/run/smbd.pid
+PIDFile=/run/samba/smbd.pid
 LimitNOFILE=16384
 EnvironmentFile=-/etc/sysconfig/samba
 ExecStart=/usr/sbin/smbd --foreground --no-process-group $SMBDOPTIONS
--- samba-4.7.4/packaging/systemd/winbind.service.0001~	2017-12-30 20:52:42.498002142 +0100
+++ samba-4.7.4/packaging/systemd/winbind.service	2017-12-30 20:55:44.029925078 +0100
@@ -5,7 +5,7 @@ After=syslog.target network.target nmb.s
 [Service]
 Type=notify
 NotifyAccess=all
-PIDFile=/run/winbindd.pid
+PIDFile=/run/samba/winbindd.pid
 EnvironmentFile=-/etc/sysconfig/samba
 ExecStart=/usr/sbin/winbindd --foreground --no-process-group "$WINBINDOPTIONS"
 ExecReload=/usr/bin/kill -HUP $MAINPID
