--- samba-4.1.4/source3/wscript_build.tirpc~	2014-02-18 20:30:42.011477913 +0100
+++ samba-4.1.4/source3/wscript_build	2014-02-18 20:32:11.737474240 +0100
@@ -982,6 +982,7 @@ bld.SAMBA3_LIBRARY('smbd_base',
                     netapi
 		    NDR_IOCTL
                     ''' + bld.env['dmapi_lib'],
+		    ldflags='-ltirpc',
                     private_library=True,
                     vars=locals())
 
